modules/operational-insights/workspace/main.bicep

```diff
@@ -73,11 +73,8 @@ param publicNetworkAccessForIngestion string = 'Enabled'
 ])
 param publicNetworkAccessForQuery string = 'Enabled'
 
-@description('Optional. Enables system assigned managed identity on the resource.')
-param systemAssignedIdentity bool = false
-
-@description('Optional. The ID(s) to assign to the resource.')
-param userAssignedIdentities object = {}
+@description('Optional. The managed identity definition for this resource.')
```

Comment:
I wonder if we should mention here that it can only be either system assigned OR user-assigned

Reply:
Added it to all modules like this (commit https://github.com/Azure/ResourceModules/pull/4124/commits/eb53c23b5848180c48399a826001d861f0588bfc). If you have suggestions regarding the wording, please let me know.